hosts_example.yml

```diff
@@ -301,6 +301,43 @@ kafka_connect:
 #     syslog:
 #     elastic:
 
+#### Connectors and the Confluent Hub
+#
+## Adding Connector Paths
+# vars:
+#   kafka_connect_plugins_path: /usr/share/java,/my/connectors/dir
+#
+## Enabling/Disabling Confluent Hub Install
+# vars:
+#   kafka_connect_confluent_hub_install: false (Defaults to true)
+#
+## Controling where the Confluent Hub is installed from
```

Comment:
Just a thought, what if we dont expose all the customizable variables? I think its great these vars exist, but i'd rather document the simplest set of thse